src/tsickle.ts

```diff
@@ -375,21 +406,39 @@ class Annotator extends ClosureRewriter {
       sourceMap: annotated.sourceMap,
     };
   }
+
+  private formatExportingDecoratorExterns() {
+    if (this.exportingDecoratorExterns.length === 0) {
+      return '';
+    } else {
+      return '\n' + this.exportingDecoratorExterns.map(name => `var ${name};\n`).join('');
```

Comment:
It seems to me that, whatever I do here, this results in the name to be "blessed" and treated magically by the Closure compiler. At least, I did not see any particular differences there.